src/app/mobile-inspect/MobileInspect.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import { useSubscription } from 'app/utils/hooks';
+import { DimItem } from 'app/inventory/item-types';
+import ItemActions from 'app/item-popup/ItemActions';
+import BungieImage from 'app/dim-ui/BungieImage';
+import { ItemSubHeader } from 'app/item-popup/ItemSubHeader';
+import { showMobileInspect$ } from './mobile-inspect';
+
+import styles from './MobileInspect.m.scss';
+import ItemSockets from 'app/item-popup/ItemSockets';
+
+export default function MobileInspect() {
+  const [item, setItem] = useState<DimItem | undefined>();
+  // TODO: In some very rare cases the popup doesn't close. Allow tapping to reset/close.
+  const reset = () => setItem(undefined);
+
+  useEffect(() => {
+    document.body.classList.toggle('mobile-preview', Boolean(item));
+    document.body.classList.toggle('mobile-preview-fade', !item);
+  }, [item]);
```

Comment:
wish there was a nice way to apply an animation when a class is removed.. In this case I could also just put up a large square/transparent div behind the popup instead of making the body transparent. I may experiment with that instead of polluting the body class.